dask/dataframe/tests/test_io.py

```diff
@@ -37,14 +38,26 @@
 Dennis,400
 Edith,-500
 Frank,600
+Alice,200
+Frank,-200
+Bob,600
+Alice,400
+Frank,200
+Alice,300
+Edith,600
 """.strip()
 
 
 @pytest.mark.parametrize('open_comp_pair,infer',
                          list(product([(open, None), (GzipFile, 'gzip'),
-                                       (BZ2File, 'bz2')], (True, False))))
+                                       (BZ2File, 'bz2'), (LZMAFile, 'xz')],
+                                      (True, False))))
 def test_read_csv(open_comp_pair, infer):
     myopen, compression = open_comp_pair
+    if compression == 'xz':
+        if LooseVersion(pd.__version__) <= '0.18.0':
+            pytest.skip()
+        pytest.importorskip('lzma')
```

Comment:
The one nice thing about doing it this way is that it shows up as skipped in py.test, instead of just not getting run. But yeah, does clutter the test a bit.